Promact.Oauth.Server/src/Promact.Oauth.Server/appsettings.development.example.json

```diff
@@ -1,11 +1,15 @@
 ﻿{
-
-  "EmailCredential": {
+  "AppSettings": {
```

Comment:
All details are kept in single block. where as aspnet core provides a much better way to manage this now. please investigate how this can be better organized.